pkgs/build-support/flatten-references-graph/src/flatten_references_graph/flatten_references_graph.py

```diff
@@ -0,0 +1,45 @@
+from toolz import curried as tlz
+
+from .lib import (
+    flatten,
+    over,
+    references_graph_to_igraph
+)
+
+from .pipe import pipe
+
+MAX_LAYERS = 127
+
+
+def create_list_of_lists_of_strings(deeply_nested_lists_or_dicts_of_graphs):
```

Comment:
This functions name seems much more generic than what it is. It expects a specific kind of things, and what it returns has a meaning other than being a `list_of_lists_of_strings`. So maybe we need a better name and some docs here? Or better, we can type the objects it takes and returns, so it looks like (not sure about what it actually does so I just made something up)

```
# Takes a possibly deeply nested list or dict containing graphs, and returns a list of
# layers.
def graphs_to_layers(graphs) -> List<List<Layer>>
```